### PR TITLE
Support DDS in project_summary

### PR DESCRIPTION
**Pull request by chuan-wang**

### CODE REVIEW

**Review by ssjunnebo — APPROVED** [review]
Very nice 👍

File: data/report_templates/project_summary.md [hunk 1]
```diff
@@ -190,6 +190,17 @@ Generation sequence Cluster Storage, [uppmax.uu.se](http://www.uppmax.uu.se)) ca
 - [NGI data delivery note](https://ngisweden.scilifelab.se/info/Data%20delivery)
 - [UPPMAX GRUS user guide](https://www.uppmax.uu.se/support/user-guides/grus-user-guide/)
 
+{% elif project.cluster == 'dds' -%}
+## Data access at DDS
+
+Data from the sequencing have been be uploaded to the **Data Delivery System**
```
Comment (aanil) [inline data/report_templates/project_summary.md:196]:
```suggestion
Data from the sequencing have been uploaded to the **Data Delivery System**
```

Comment (aanil) [reply data/report_templates/project_summary.md:196]: I guess the text for grus should be updated as well?

Comment (chuan-wang) [reply data/report_templates/project_summary.md:196]: Sorry I just copied without reading :D

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**